shell/platform/darwin/ios/framework/Source/flutter_main_ios.mm

```diff
@@ -12,7 +12,13 @@ void FlutterMain() {
   NSBundle* bundle = [NSBundle bundleForClass:[FlutterViewController class]];
   NSString* icuDataPath = [bundle pathForResource:@"icudtl" ofType:@"dat"];
   NSString* libraryName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"FLTLibraryPath"];
-  shell::PlatformMacMain(icuDataPath.UTF8String, libraryName != nil ? libraryName.UTF8String : "");
+
+  NSBundle* mainBundle = [NSBundle mainBundle];
+
+  NSString* flutterAssetsPath = [mainBundle pathForResource:@"flutter_assets" ofType:nil];
```

Comment:
This should check for an override at the FLTAssetsPath key in the plist.

NSString* flutterAssetsName = [mainBundle objectForInfoDictionaryKey:@"FLTAssetsPath"];
if (flutterAssetsName == nil) {
   // Default to "flutter_assets"
flutterAssetsName = @"flutter_assets";
}

Reply:
Sorry about that, thanks @cbracken !